src/student/dto/schedule-monitoring.dto.ts

```diff
@@ -13,4 +14,12 @@ export class ScheduleMonitoringDto {
   })
   @Transform(({ value }) => value && new Date(value))
   end: Date;
+
+  @ApiProperty({
+    example: 'Aula Sobre Vetores e Matrizes em AED I',
+  })
+  @IsOptional()
```

Comment:
Aqui você pode usar deocrators para fazer as validações no campo:
```suggestion
  @ApiProperty({
    example: 'Aula Sobre Vetores e Matrizes em AED I',
  })
  @IsString(500)
  @MaxLength(500)
  @IsOptional()
```

Os dois decorators novos são importados da lib `class-validator`.